tests/jenkins/test_loading.py

```diff
@@ -60,8 +61,8 @@
         ),
     ],
 )
-def test_load_data_from_csv(tmp_path, test_data_path, csv_file, config, exc):
-    csv_path = test_data_path / "jenkins" / "validation" / csv_file
+def test_load_data_from_csv(tmp_path, test_data_path, pathogen, csv_file, config, exc):
+    csv_path = test_data_path / "integration_test_validation" / pathogen / csv_file
```

Comment:
A lot of file changes below are due to this change in location as we need to differentiate per pathogen.